### DISCUSSION TITLE
Sourcing the SMD Dual Battery Holder

### DISCUSSION THREAD

**Discussion by jbatesmd** [alanbjohnston/CubeSatSim#314] [General]
I have been unable to find the dual SMD battery holder from the major distributors.  I currently have it backordered from Digikey.  If someone has a spare, I would gladly pay for it or swap it out when mine comes in.  I have built the payload board and all seems to be fine.  This is a great project and the instructions are easy to follow and spot on.

**Comment by alanbjohnston** [comment]
Hi,

Yes, the BT2 dual AA battery holder (Keystone 1012) appears to be out of stock everywhere.  Fortunately, you can just order two additional BT1 single AA holders and use them (part Keystone 1024).  I have updated this in the Octopart BOM.

The only modification you will need to make is to cut or remove the little black plastic alignment tabs on the bottom of the BT1.  Both will need the tab removed so it lays flat. Make sure that there is room for the M2.5 standoff in the lower right hole.

Good luck with your build!

Alan

**Reply by tourbrown** [reply]
In the picture it looks as though both of the bottom holders are reversed from the board layout...

The text mentioned making sure the bottom battery holder was reversed but did not mention the middle one.

Is that also reversed?

**Comment by alanbjohnston** [comment]
You are right!   The photo showed both of them backwards (I deleted the photo).  Here's how they should look:

![image](https://github.com/alanbjohnston/CubeSatSim/assets/25288070/a8210b9b-8d1f-441d-8ca9-3f4ca25fc5fb)

The holder is actually symmetric, but having the + and - labels correct is a good idea.

Thanks for pointing it out!

Alan
